src/components/StudyCard/StudyCard.style.ts

```diff
@@ -55,7 +70,7 @@ export const Ul = styled.ul`
 export const RightColumn = styled.div`
   flex-shrink: 0;
   width: 296px;
-  height: 208px;
+  height: 100%;
 
   @media (max-width: 568px) {
     width: 100%;
```

Comment:
```suggestion
    display: none;
```
이미지보단 스터디 내용이 보이는게 좋은 것 같습니다! 미디어 쿼리를 잘 몰라서 이게 맞는지는 모르겠네요.. 죄송합니다😓
![image](https://user-images.githubusercontent.com/66072832/183868765-392e686a-88dd-4a1b-a5cb-9377bb2a3ade.png)